### PR TITLE
How-to:/deploy/tiup.md for deploy tiflash

### PR DESCRIPTION
**Pull request by superlzs0476**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by superlzs0476** [comment]
@lilin90  @ilovesoup  @cocoyi17  PTAL

File: how-to/deploy/orchestrated/tiup.md [hunk 1]
```diff
@@ -310,15 +310,20 @@ category: how-to
...
+| Tiflash | 1 | 32 VCore 64 GB * 1 | 10.0.1.10 | 默认端口 <br> 全局目录配置 |
 
 #### 第 4 步：配置文件模版 topology.yaml
 
 > **注意：**
 >
-> 无需手动创建 tidb 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
+> - 无需手动创建 tidb 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
+> 
+> - [部署 Tiflash](https://pingcap.com/docs-cn/dev/reference/tiflash/deploy/) 需要开启 `replication.enable-placement-rules` ，以开启 PD 的 [Placement Rules](https://pingcap.com/docs-cn/dev/how-to/configure/placement-rules/) 功能。
```
Comment (marsishandsome) [inline how-to/deploy/orchestrated/tiup.md:324]: 同一个文档内部链接是不是用 `/reference/tiflash/deploy` 就可以了？

File: how-to/deploy/orchestrated/tiup.md [hunk 2]
```diff
@@ -403,6 +409,9 @@ tikv_servers:
     #      host: host1
   - host: 10.0.1.2
   - host: 10.0.1.3
+tiflash_servers:
+  - host: 10.0.1.10
+    # data_dir: /data1/tiflash/data,/data2/tiflash/data
```
Comment (marsishandsome) [inline how-to/deploy/orchestrated/tiup.md:414]: 其他组件的配置的注释好像都是从 https://github.com/pingcap-incubator/tiup-cluster/blob/master/topology.example.yaml 拷贝过来

tiflash是不是也可以拷贝过来？
```
 # ssh_port: 22
    # tcp_port: 9000
    # http_port: 8123
    # flash_service_port: 3930
    # flash_proxy_port: 20170
    # flash_proxy_status_port: 20292
    # metrics_port: 8234
    # deploy_dir: deploy/tiflash-9000
    # data_dir: deploy/tiflash-9000/data
    # log_dir: deploy/tiflash-9000/log
    # numa_node: "0,1"
    # # Config is used to overwrite the `server_configs.tiflash` values
    #  config:
    #    logger:
    #      level: "info"
    #  learner_config:
    #    log-level: "info"
```

File: how-to/deploy/orchestrated/tiup.md (hunk 1 above)
Comment (yikeke) [reply how-to/deploy/orchestrated/tiup.md:324]: 是的，内链统一用相对链接哈，方便 cherry-pick： @superlzs0476 
- https://pingcap.com/docs-cn/dev/reference/tiflash/deploy/ -> /reference/tiflash/deploy.md
- https://pingcap.com/docs-cn/dev/how-to/configure/placement-rules/ -> /how-to/configure/placement-rules.md

File: how-to/deploy/orchestrated/tiup.md (hunk 2 above)
Comment (superlzs0476) [reply how-to/deploy/orchestrated/tiup.md:414]: 提供一般参数介绍吧，这么多参数，Tiflash 的部署文档里面都没有。

File: how-to/deploy/orchestrated/tiup.md (hunk 1 above)
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:324]:
```suggestion
> - [部署 Tiflash](https://pingcap.com/docs-cn/dev/reference/tiflash/deploy/) 需要在 topology.yaml 配置文件中将 `replication.enable-placement-rules` 设置为 `true`，以开启 PD 的 [Placement Rules](https://pingcap.com/docs-cn/dev/how-to/configure/placement-rules/) 功能。
```

Comment (superlzs0476) [reply how-to/deploy/orchestrated/tiup.md:324]: 没明白，你来提一个 suggestion 我来点赞吧

File: how-to/deploy/orchestrated/tiup.md [hunk 3]
```diff
@@ -476,15 +485,21 @@ alertmanager_servers:
...
+| Tiflash | 1 | 32 VCore 64 GB | 10.0.1.10 | 默认端口 <br> 自定义部署目录 /data1/tiflash/data |
 
 #### 第 4 步：配置文件模版 topology.yaml
 
 > **注意：**
 > 
-> 配置文件模版时，注意修改必要参数、IP、端口及目录。
+> - 配置文件模版时，注意修改必要参数、IP、端口及目录。
+> 
+> - [部署 Tiflash](https://pingcap.com/docs-cn/dev/reference/tiflash/deploy/) 需要开启 `replication.enable-placement-rules` ，以开启 PD 的 [Placement Rules](https://pingcap.com/docs-cn/dev/how-to/configure/placement-rules/) 功能。
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:499]:
```suggestion
> - [部署 Tiflash](https://pingcap.com/docs-cn/dev/reference/tiflash/deploy/) 需要在 topology.yaml 配置文件中将 `replication.enable-placement-rules` 设置为 `true`，以开启 PD 的 [Placement Rules](https://pingcap.com/docs-cn/dev/how-to/configure/placement-rules/) 功能。
```

File: how-to/deploy/orchestrated/tiup.md (hunk 2 above)
Comment (marsishandsome) [inline how-to/deploy/orchestrated/tiup.md:414]:
```suggestion
    # ssh_port: 22
    # tcp_port: 9000
    # http_port: 8123
    # flash_service_port: 3930
    # flash_proxy_port: 20170
    # flash_proxy_status_port: 20292
    # metrics_port: 8234
    # deploy_dir: deploy/tiflash-9000
    # data_dir: deploy/tiflash-9000/data
    # log_dir: deploy/tiflash-9000/log
    # numa_node: "0,1"
    # # Config is used to overwrite the `server_configs.tiflash` values
    #  config:
    #    logger:
    #      level: "info"
    #  learner_config:
    #    log-level: "info"
```

File: how-to/deploy/orchestrated/tiup.md [hunk 4]
```diff
@@ -476,15 +485,21 @@ alertmanager_servers:
 
 | 实例 | 个数 | 物理机配置 | IP | 配置 |
 | :-- | :-- | :-- | :-- | :-- |
-| TiKV | 6 | 32 Vcore 64GB * 3 | 10.0.1.1<br> 10.0.1.2<br> 10.0.1.3 | 1. 区分实例级别的 port、status_port；<br> 2. 配置全局参数 readpool、storage 以及 raftstore 参数；<br> 3. 配置实例级别 host 维度的 labels；<br> 4. 配置 numa 绑核操作|
-| TiDB | 6 | 32 Vcore 64GB * 3 | 10.0.1.7<br> 10.0.1.8<br> 10.0.1.9 | 配置 numa 绑核操作 |
-| PD | 3 | 16 Vcore 32 GB | 10.0.1.4<br> 10.0.1.5<br> 10.0.1.6 | 配置 location_lables 参数 |
+| TiKV | 6 | 32 VCore 64GB | 10.0.1.1<br> 10.0.1.2<br> 10.0.1.3 | 1. 区分实例级别的 port、status_port；<br> 2. 配置全局参数 readpool、storage 以及 raftstore 参数；<br> 3. 配置实例级别 host 维度的 labels；<br> 4. 配置 numa 绑核操作|
+| TiDB | 6 | 32 VCore 64GB | 10.0.1.7<br> 10.0.1.8<br> 10.0.1.9 | 配置 numa 绑核操作 |
+| PD | 3 | 16 VCore 32 GB | 10.0.1.4<br> 10.0.1.5<br> 10.0.1.6 | 配置 location_lables 参数 |
+| Tiflash | 1 | 32 VCore 64 GB | 10.0.1.10 | 默认端口 <br> 自定义部署目录 /data1/tiflash/data |
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:492]:
```suggestion
| Tiflash | 1 | 32 VCore 64 GB | 10.0.1.10 | 默认端口 <br> 自定义部署目录，配置 data_dir 参数为 `/data1/tiflash/data` |

```

File: how-to/deploy/orchestrated/tiup.md [hunk 5]
```diff
@@ -653,19 +670,24 @@ TiDB 关键参数：
...
-|TiDB | 16 vcore 32 GB * 3 | 10.0.1.7 <br> 10.0.1.8 <br> 10.0.1.9 | 默认端口配置；<br>开启 enable_binlog； <br> 开启 ignore-error |
-| PD | 4 vcore 8 GB * 3| 10.0.1.4 <br> 10.0.1.5 <br> 10.0.1.6 | 默认端口配置 |
-| Pump|8 vcore 16GB * 3|10.0.1.6<br>10.0.1.7<br>10.0.1.8 | 默认端口配置； <br> 设置 GC 时间 7 天 |
-| Drainer | 8 vcore 16GB | 10.0.1.9 | 默认端口配置；<br>设置默认初始化 commitTS |
+| 实例 |个数| 物理机配置 | IP | 配置 |
+| :-- | :-- | :-- | :-- | :-- |
+| TiKV | 3 | 16 VCore 32 GB | 10.0.1.1 <br> 10.0.1.2 <br> 10.0.1.3 | 默认端口配置 |
+|TiDB | 3 | 16 VCore 32 GB | 10.0.1.7 <br> 10.0.1.8 <br> 10.0.1.9 | 默认端口配置；<br>开启 enable_binlog； <br> 开启 ignore-error |
+| PD | 3 | 4 VCore 8 GB | 10.0.1.4 <br> 10.0.1.5 <br> 10.0.1.6 | 默认端口配置 |
+| Tiflash | 1 | 32 VCore 64 GB  | 10.0.1.10 | 默认端口 <br> 自定义部署目录，通过 /data1/tiflash/data,/data2/tiflash/data 多盘部署 |
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:678]:
```suggestion
| Tiflash | 1 | 32 VCore 64 GB  | 10.0.1.10 | 默认端口 <br> 自定义部署目录，配置 data_dir 参数为 `/data1/tiflash/data,/data2/tiflash/data`，进行多盘部署 |
```

File: how-to/deploy/orchestrated/tiup.md [hunk 6]
```diff
@@ -653,19 +670,24 @@ TiDB 关键参数：
...
+| Drainer | 1 | 8 VCore 16GB | 10.0.1.9 | 默认端口配置；<br>设置默认初始化 commitTS |
 
 #### 第 4 步：配置文件模版 topology.yaml
 
 > **注意：**
 > 
-> 配置文件模版时，如无需自定义端口或者目录，仅修改 IP 即可。
+> - 配置文件模版时，如无需自定义端口或者目录，仅修改 IP 即可。
+>
+> - [部署 Tiflash](https://pingcap.com/docs-cn/dev/reference/tiflash/deploy/) 需要开启 `replication.enable-placement-rules` ，以开启 PD 的 [Placement Rules](https://pingcap.com/docs-cn/dev/how-to/configure/placement-rules/) 功能。
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:688]:
```suggestion
> - [部署 Tiflash](https://pingcap.com/docs-cn/dev/reference/tiflash/deploy/) 需要在 topology.yaml 配置文件中将 `replication.enable-placement-rules` 设置为 `true`，以开启 PD 的 [Placement Rules](https://pingcap.com/docs-cn/dev/how-to/configure/placement-rules/) 功能。
```

File: how-to/deploy/orchestrated/tiup.md (hunk 1 above)
Comment (marsishandsome) [inline how-to/deploy/orchestrated/tiup.md:324]:
```suggestion
> - [部署 Tiflash](/reference/tiflash/deploy/) 需要开启 `replication.enable-placement-rules` ，以开启 PD 的 [Placement Rules](/how-to/configure/placement-rules/) 功能。
```

Comment (yikeke) [reply how-to/deploy/orchestrated/tiup.md:324]: 应该是 /reference/tiflash/deploy.md 哈

File: how-to/deploy/orchestrated/tiup.md [hunk 7]
```diff
@@ -310,15 +310,20 @@ category: how-to
...
+| Tiflash | 1 | 32 VCore 64 GB * 1 | 10.0.1.10 | 默认端口 <br> 全局目录配置 |
 
 #### 第 4 步：配置文件模版 topology.yaml
 
 > **注意：**
 >
-> 无需手动创建 tidb 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
+> - 无需手动创建 tidb 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
+> 
+> - [部署 Tiflash](/reference/tiflash/deploy/) 需要开启 `replication.enable-placement-rules` ，以开启 PD 的 [Placement Rules](/how-to/configure/placement-rules/) 功能。
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:324]:
```suggestion
> - [部署 Tiflash](/reference/tiflash/deploy.md) 需要在 topology.yaml 配置文件中将 `replication.enable-placement-rules` 设置为 `true`，以开启 PD 的 [Placement Rules](/how-to/configure/placement-rules.md) 功能。
```

File: how-to/deploy/orchestrated/tiup.md [hunk 8]
```diff
@@ -476,15 +501,21 @@ alertmanager_servers:
...
 
 > **注意：**
 > 
-> 配置文件模版时，注意修改必要参数、IP、端口及目录。
+> - 配置文件模版时，注意修改必要参数、IP、端口及目录。
+> 
+> - [部署 Tiflash](/reference/tiflash/deploy.md) 需要在 topology.yaml 配置文件中将 `replication.enable-placement-rules` 设置为 `true`，以开启 PD 的 [Placement Rules](/how-to/configure/placement-rules.md) 功能。
+>
+> - tiflash_servers 实例级别配置 `"-host"` 目前只支持 IP，不支持域名。
+
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:518]:
```suggestion
```
to fix CI

File: how-to/deploy/orchestrated/tiup.md [hunk 9]
```diff
@@ -324,6 +324,8 @@ category: how-to
 > - [部署 Tiflash](/reference/tiflash/deploy.md) 需要在 topology.yaml 配置文件中将 `replication.enable-placement-rules` 设置为 `true`，以开启 PD 的 [Placement Rules](/how-to/configure/placement-rules.md) 功能。
 >
 > - tiflash_servers 实例级别配置 `"-host"` 目前只支持 IP，不支持域名。
+>
+> - Tiflash 具体的参数配置介绍请见[常见部署问题 Tiflash 参数配置](#常见部署问题)。
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:328]:
```suggestion
> - Tiflash 具体的参数配置介绍请见[常见部署问题 Tiflash 参数配置](#tiflash-参数)。
```

File: how-to/deploy/orchestrated/tiup.md [hunk 10]
```diff
@@ -703,6 +707,8 @@ TiDB 关键参数：
 > - [部署 Tiflash](/reference/tiflash/deploy.md) 需要在 topology.yaml 配置文件中将 `replication.enable-placement-rules` 设置为 `true`，以开启 PD 的 [Placement Rules](/how-to/configure/placement-rules.md) 功能。
 >
 > - tiflash_servers 实例级别配置 `"-host"` 目前只支持 ip，不支持域名。
+>
+> - Tiflash 具体的参数配置介绍请见[常见部署问题 Tiflash 参数配置](#常见部署问题)。
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:711]:
```suggestion
> - Tiflash 具体的参数配置介绍请见[常见部署问题 Tiflash 参数配置](#tiflash-参数)。
```

File: how-to/deploy/orchestrated/tiup.md [hunk 11]
```diff
@@ -515,6 +517,8 @@ alertmanager_servers:
 > - [部署 Tiflash](/reference/tiflash/deploy.md) 需要在 topology.yaml 配置文件中将 `replication.enable-placement-rules` 设置为 `true`，以开启 PD 的 [Placement Rules](/how-to/configure/placement-rules.md) 功能。
 >
 > - tiflash_servers 实例级别配置 `"-host"` 目前只支持 IP，不支持域名。
+>
+> - Tiflash 具体的参数配置介绍请见[常见部署问题 Tiflash 参数配置](#常见部署问题)。
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:521]:
```suggestion
> - Tiflash 具体的参数配置介绍请见[常见部署问题 Tiflash 参数配置](#tiflash-参数)。
```

File: how-to/deploy/orchestrated/tiup.md [hunk 12]
```diff
@@ -1341,6 +1347,22 @@ Destroyed cluster `tidb-test` successfully
 | 实例 | data_dir | 继承 global 配置 | 数据目录 |
 | 实例 | log_dir | 继承 global 配置 | 日志目录 |
 
+
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:1350]:
```suggestion
```
Multiple blanks will fail the CI.

**Pull request merged**